api/projects.ts

```diff
@@ -275,36 +276,40 @@ export async function cancelStagedBuild(
   });
 }
 
-export async function fetchBuildWarnLogs(
+type WarnLogsResponse = {
+  logs: Array<ProjectLog>;
+};
```

Comment:
Do we want to move this to a types file?

Reply:
Hmm good question. For smaller types that are only used in one file, I generally just keep them in that file. Though thinking from a library perspective it would probably make sense to export all those types.

This is done in a lot of places though so would need a separate PR